README.md

```diff
@@ -45,48 +46,30 @@ Skip this step if adding the module to an existing project.
 Switch to the newly created webroot, and add the SilverStripe Behat extension.
 
 	cd my-test-project
-	composer require --dev silverstripe/behat-extension:"@stable"
+	composer require --dev silverstripe/behat-extension
 
-Now get the latest Selenium2 server (requires [Java](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)):
+Download the standalone [Google Chrome WebDriver](http://chromedriver.storage.googleapis.com/index.html?path=2.8/)
 
-	composer require --dev se/selenium-server-standalone:"2.x@stable"
-
-Download [Firefox 31.0 ESR](https://ftp.mozilla.org/pub/firefox/releases/31.8.0esr/) (Extended Support Release).
-This version is older than your currently installed Firefox.
-It's important to have a browser that's [supported by Selenium-Webdriver](http://docs.seleniumhq.org/docs/01_introducing_selenium.jsp#selenium-webdriver). Even newer Firefox ESR versions are likely to break with the Selenium version we're running.
 
 Now install the SilverStripe project as usual by opening it in a browser and following the instructions.
 Protip: You can skip this step by using `[SS_DATABASE_CHOOSE_NAME]` in a global
-[`_ss_environment.php`](http://doc.silverstripe.org/framework/en/topics/environment-management)
-file one level above the webroot.
+[`.env`](https://docs.silverstripe.org/en/getting_started/environment_management/) file (or [`_ss_environment.php`](https://docs.silverstripe.org/en/3/getting_started/environment_management/)
+file for SilverStripe core versions 3.x and under) one level above the webroot.
 
-Unless you have [`$_FILE_TO_URL_MAPPING`](http://doc.silverstripe.org/framework/en/topics/commandline#configuration)
-set up, you also need to specify the URL for your webroot. Either add it to the existing `behat.yml` configuration file
+Unless you have [`SS_BASE_URL`](http://doc.silverstripe.org/framework/en/topics/commandline#configuration) (or 
+[`$_FILE_TO_URL_MAPPING`](https://docs.silverstripe.org/en/3/developer_guides/cli/#configuration) respectively) set up, 
+you also need to specify the URL for your webroot. Either add it to the existing `behat.yml` configuration file
```

Comment:
You can probably remove the SS3 related content from this branch since it requires SS4